codec/codec.go

```diff
@@ -57,6 +57,7 @@ const (
 	VEDITKey                     = "VEDIT"
 	OutputAddressEditKey         = "OEDIT"
 	ClearUnjailedValSessionKey   = "CRVAL"
+	AppTransferKey               = "AppTransfer"
```

Comment:
This may be bikeshedding.  Do we need to keep feature names cryptic codenames like those?  In my understanding, there is no limitation in case sensitivity or the length.  I think having more descriptive names makes the result of `/v1/query/upgrade` easier to understand.

Reply:
Not adamant either way. Feel free to resolve.